templates/repo/view_file.tmpl

```diff
@@ -48,6 +48,7 @@
 				{{end}}
 				<a class="ui mini basic button" href="{{.RepoLink}}/commits/{{EscapePound .BranchNameSubURL}}/{{EscapePound .TreePath}}">{{.i18n.Tr "repo.file_history"}}</a>
 			</div>
+			<a download href="{{EscapePound $.RawFileLink}}"><span class="btn-octicon poping up" data-content="{{.i18n.Tr "repo.download_file"}}" data-position="bottom center" data-variation="tiny inverted">{{svg "octicon-download"}}</span></a>
```

Comment:
The usage of `EscapePound` seems wrong here.

Reply:
I've copied the href part from the raw button, so it should be good.

Reply:
Oh I see the EscapePound above. The EscapePound itself is doing wrong 😂  I just remember zeripath submitted a PR to fix them.